### ISSUE TITLE
ImageMagick pixel enumeration parser

### ISSUE THREAD

**Issue by rexim** [codingteam/pacmacs.el#216]
Is part of #41

Maybe it's gonna be a utility or something

**Comment by rexim** [comment]
I just found a nice little feature of ImageMagick. If you invoke convert like `$ convert <image> txt:-` it will dump to stdin info about image pixels

```
# ImageMagick pixel enumeration: 1,3,255,rgb
0,0: ( 38,209,145)  #26D191  rgb(38,209,145)
0,1: ( 31,215,172)  #1FD7AC  rgb(31,215,172)
0,2: ( 51,194, 77)  #33C24D  rgb(51,194,77)
```

which is much easier to parse with Emacs Lisp than XPM.

I think we can use that feature as an intermediate step for generating sprites in The Infernal Format.

**Comment by rexim** [comment]
The scope of the issue has changed. Now we need only a parser for ImageMagick pixel enumeration format.

**Comment by rexim** [comment]
Closed in 5cf1b858b870fe71e48000d8679fd7487d786288
